bigbluebutton-web/grails-app/controllers/org/bigbluebutton/web/controllers/ApiController.groovy

```diff
@@ -181,7 +181,7 @@ class ApiController {
         Meeting existingWebVoice = meetingService.getNotEndedMeetingWithWebVoice(newMeeting.getWebVoice());
         if (existingTelVoice != null || existingWebVoice != null) {
           log.error "VoiceBridge already in use by another meeting (different meetingId)"
-          errors.nonUniqueMeetingIdError()
+          errors.nonUniqueVoiceBridgeError()
```

Comment:
Would it make sense to pass the voice bridge in the error so we can log it (for easier debugging)?

Reply:
Sure. Is there a pattern for log filtering in bbb-web? Would only the `voiceBridge` be enough?

Reply:
I am not aware of a pattern. I would use `noConfigFoundForToken` as example

```
	public void noConfigFoundForToken(String token) {
		errors.add(new String[] {"configNotFound", "We could not find a config for token [" + token + "]."});
	}
```

https://github.com/bigbluebutton/bigbluebutton/blob/de40885768efe7c27695082609aa177081eaef0b/bbb-common-web/src/main/java/org/bigbluebutton/api/ApiErrors.java#L64

Reply:
Oh, I though you were talking about the `log.error` message. I'm not sure if we should supply the voice bridge value to the xml response since is a direct response to the API caller. We sure don't do that in most of the other error responses.